langchain4j-redis/src/main/java/dev/langchain4j/store/embedding/redis/RedisEmbeddingStore.java

```diff
@@ -259,14 +268,16 @@ public Builder password(String password) {
             return this;
         }
 
+        
         /**
-         * @param indexName The name of the index (optional). Default value: "embedding-index".
+         * @param name The name of the store (optional). Used for prefix and index name generation. 
+         * Default value: "embedding".
          * @return builder
          */
-        public Builder indexName(String indexName) {
```

Comment:
I am pretty sure there are users that use this method, so could you please make this change in a non-breaking way? You could keep `indexName` method and mark is as `@Deprecated` and introduce a new one called `name`

Reply:
Hi @langchain4j , good point. 

I just sent the update. I went with deprecated indexName not setting any field, assuming that accessing already created embeddings (with default prefix) have higher priority than keeping index name intact. I view index as a part of internal storage representation. 

With that approach, embeddings already created with langchain4j, would be accessible without problems after fix is applied. Although additional embedding index would be created in the Redis instance.

I was also thinking about exposing indexName as additional parameter, so user of RedisEmbeddingStore could specify separately prefix for keys and indexName. But it would make creation of RedisEmbeddingStore a bit more complicated, so for now I stayed with simpler approach.

@langchain4j , please let me know, if this approach looks good to you.